src/liballoc/sync.rs

```diff
@@ -29,6 +28,26 @@ use crate::rc::is_dangling;
 use crate::string::String;
 use crate::vec::Vec;
 
+// Since the current `liballoc` is not a superset of `libcore`,
+// using `pub use core::sync::atomic;` will fail with some link errors.
+// If `liballoc` becomes a superset of `libcore`, replace this with
+// `pub use core::sync::atomic;`.
```

Comment:
I added a description.

Reply:
Specifically, it is this error: https://github.com/rust-lang/rust/pull/58175#issuecomment-460960366

```text
[01:36:43] alloc/sync/atomic/index.html:13: broken link - alloc/marker/trait.Sync.html
```